### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by oigroigm** [Tomcuzz/homebridge-tasmota-sonoff-thermostat#2]
Is it possible to enter the possibility of adjusting the humidity and temperature offset to calibrate the sensor ?

**Comment by Tomcuzz** [comment]
Yep this is an awesome idea and happy to add this, although have some features that Im working on first (pulling through humidity and ability to link in with google home/home assistant). Feel free to put in a pull request if you want this added urgently

**Comment by Tomcuzz** [comment]
Updating title
